commands/create_keypair.go

```diff
@@ -42,79 +89,138 @@ func init() {
 	CreateCommand.AddCommand(CreateKeypairCommand)
 }
 
-func checkAddKeypair(cmd *cobra.Command, args []string) error {
+func createKeyPairPreRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
+	args := defaultCreateKeypairArguments()
+	err := verifyCreateKeypairPreconditions(args)
 
-	endpoint := config.Config.ChooseEndpoint(cmdAPIEndpoint)
-	token := config.Config.ChooseToken(endpoint, cmdToken)
+	if err == nil {
+		return
+	}
```

Comment:
In other function we have

```
+	if err != nil {
+		handleCommonErrors(err)
```

Even though I'm **huge** fan of fast returns, in this case, this is possible there will be something added forcing use of additional indentation and obscuring the diff. I think unifying that would be beneficial.

Reply:
I still think this comment is valid.

Reply:
Still thinking about doing the alignment...

Reply:
I would actually rather align to fast returns in all commands, as it would decrease the complexity of the code involved.